From 5e80e02c2b2ec733ff645575686e642e4cc69bf6 Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Sat, 25 May 2013 08:31:25 +0200
Subject: [PATCH 47/48] DOC: fix wrong copy-paste in the rspdel example

As reported by Cristian Ditoiu, the rspdel example is based on the reqrep
action, which is quite confusing!
---
 doc/configuration.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/configuration.txt b/doc/configuration.txt
index 9fed5c8..4561741 100644
--- a/doc/configuration.txt
+++ b/doc/configuration.txt
@@ -5011,7 +5011,7 @@ rspidel <search> [{if | unless} <cond>]  (ignore case)
 
   Example :
      # remove the Server header from responses
-     reqidel ^Server:.*
+     rspidel ^Server:.*
 
   See also: "rspadd", "rsprep", "reqdel", section 6 about HTTP header
             manipulation, and section 7 about ACLs.
-- 
1.8.1.5

